sdk/nodejs/examples/pastebin/src/index.ts

```diff
@@ -0,0 +1,82 @@
+const { Command } = require("commander");
+const zrok = require("zrok")
+const ziti =  require('@openziti/ziti-sdk-nodejs')
+var readlineSync = require('readline-sync');
+
+
+const program = new Command();
+
+program
+  .command('copyto')
+  .version("1.0.0")
+  .description("command to host content to be pastedfrom'd")
+  .action(async () => {
+    var data = readlineSync.question('Input some text... ');
+
+    console.log("data is: ", data)
+    let root = zrok.Load()
+    await zrok.init( root ).catch(( err: Error ) => { console.error(err); return process.exit(1) });
+    zrok.setLogLevel(0)
+    console.log("setting up zrok.CreateShare...")
+    let shr = await zrok.CreateShare(root, new zrok.ShareRequest(zrok.TCP_TUNNEL_BACKEND_MODE, zrok.PRIVATE_SHARE_MODE, "pastebin", ["private"]));
+    console.log("access your pastebin using 'pastefrom ", shr.Token)
+    let app = zrok.express( shr.Token );
+    app.get('/',function(_: Request,res: any){
```

Comment:
The `copyto` and `pastefrom` in the other languages don't use HTTP, they use raw sockets. The node implementations should interoperate with the golang or python versions. Is this the case?

If so, and we're using a plain TCP socket, why are we using `express`?